packages/puppeteer-core/src/common/bidi/Frame.ts

```diff
@@ -275,7 +193,7 @@ export class Frame extends BaseFrame {
     return this.#detached;
   }
 
-  dispose(): void {
+  [Symbol.dispose](): void {
```

Comment:
I think this should not be changed as is user facing. I think user facing changes should remain with `dispose()`.

Reply:
I don't think this is user-facing. Frames can't be disposed by the user.